src/toast/ops/stokes_weights.py

```diff
@@ -168,6 +168,23 @@ def _exec(self, data, detectors=None, **kwargs):
                 # Nothing to do for this observation
                 continue
 
+            # Create (or re-use) output data for the weights
+
+            if self.single_precision:
+                existed = ob.detdata.ensure(
```

Comment:
This PR uses `exists` and `existed` for the return value. Are we happy with that or should we pick one and use it consistently?

Reply:
Good point, changed to "exists" everywhere.